src/core/Slider/Slider.tsx

```diff
@@ -0,0 +1,454 @@
+};
+
+const formatNumberValue = (
+  value: number,
+  step: number,
+  numDecimals: number,
+) => {
+  if (Number.isInteger(step)) {
+    return value.toFixed(0);
+  }
+  return value.toFixed(numDecimals);
+};
+
+/**
+ * Properties for Slider component
+ */
+export type SliderProps = {
+  /**
+   * Set focus on first thumb in slider element.
+   * @default false
+   */
+  setFocus?: boolean;
+  /**
+   * Minimum slider value.
+   * @default 0
+   */
+  min?: number;
+  /**
+   * Maximum slider value.
+   * @default 100
+   */
+  max?: number;
+  /**
+   * Array of one or more values to show.
+   */
+  values: number[];
+  /**
+   *  Determines which segments are shown with color.
+   * 'none' - no colored tracks are displayed.
+   * 'auto' - no colored tracks are displayed.
+   * 'odd-segments'- colored tracks shown in segments 1,3,5, etc.
+   *                 Default if number of thumbs values are even.
+   * 'even-segments'- colored tracks shown in segments 0,2,4, etc.
+   *                 Default if number of thumbs values are odd.
+   * @default 'auto'
+   */
+  trackDisplayMode?: TrackDisplayMode;
+  /**
+   * Step increment controls what values are allowed and the amount the value will
+   * change when left and right arrows are pressed when a Thumb has focus.
+   * @default 1
+   */
+  step?: number;
+  /**
+   * Forces control to be displayed in a disabled state where no interactive value
+   * changes are allowed.
+   *  @default false
+   */
+  disabled?: boolean;
+  /**
+   * Props to override default for showing a tooltip when Thumb is active or has focus.
+   * changes are allowed.
+   */
+  tooltipProps?: Partial<Omit<TooltipProps, 'content' | 'children'>>;
+  /**
+   * Optional tooltip function that can be supplied to generate tooltip text.
+   */
+  tooltipRenderer?: (val: number, step: number) => React.ReactNode;
+  /**
+   *  Optional array of labels that will be used to determine number of ticks
+   * displayed and their labels. Ticks are spaced evenly across width of Slider.
+   */
+  tickLabels?: React.ReactNode[];
+  /**
+   * Optional label for the minimum value. If undefined then the min
+   * value is shown. Use empty string for no label.
+   */
+  minLabel?: React.ReactNode;
+  /**
+   * Optional label for the maximum value. If undefined then the max
+   *  value is shown. Use empty string for no label.
+   */
+  maxLabel?: React.ReactNode;
+  /**
+   * Defines the allowed behavior when moving Thumbs when multiple Thumbs are
+   * shown. It controls if a Thumb movement should be limited to only move in
+   * the segments adjacent to the Thumb. Possible values:
+   * 'allow-crossing' - allows thumb to cross other thumbs. Default.
+   * 'inhibit-crossing'- keeps the thumb from crossing and separated by a step.
+   * @default 'inhibit-crossing'
+   */
+  thumbMode?: 'allow-crossing' | 'inhibit-crossing';
+  /**
+   * Additional props to include on each thumb container.
+   */
+  thumbProps?: Omit<CommonProps, 'title'>;
```

Comment:
I wonder if there was a need to have different props for different thumbs....

Reply:
Similar comment: https://github.com/iTwin/iTwinUI-react/pull/197#discussion_r669116634